src/components/Dropdown/Dropdown.tsx

```diff
@@ -18,6 +18,8 @@ import React, { FC, ReactElement, useEffect, useRef } from "react";
 import { DEFAULT_DROPDOWN_MAX_HEIGHT, useDropdownAutoHeight } from "./useDropdownAutoHeight";
 import { Validation } from "@utilities/validation";
 import { LoadingCircle, LoadingCircleSize } from "@components/LoadingCircle";
+import { createPortal } from "react-dom";
+import { usePopper } from "react-popper";
```

Comment:
By adding VariationPlacement we get access to the placement types.

```suggestion
import { usePopper } from "react-popper";
import { VariationPlacement } from "@popperjs/core";
```